### ISSUE TITLE
ESP32: cannot build firmware any more

### ISSUE THREAD

**Issue by robert-hh** [micropython/micropython#6644]
After pulling today's PRs I cannot build the firmware for ESP32 any more. Error message:
```
GEN build-GENERIC_SPIRAM/genhdr/qstr.i.last
../../extmod/nimble/modbluetooth_nimble.c:47:44: fatal error: nimble/host/src/ble_l2cap_priv.h: No such file or directory
compilation terminated.
```
The file exists under $(TOP)/lib/mynewt-nimble So it looks like the include path for that step is wrong.

**Comment by jimmo** [comment]
Thanks @robert-hh -- should be fixed by #6645 

I will investigate why the CI missed that.

**Comment by robert-hh** [comment]
Ok. It build now and runs.

**Comment by dpgeorge** [comment]
Fixed by 5a7027915c3b9ec2dffc3af9530dad90b56b8cc0
